backstory/src/demo/webapp/perspective.js

```diff
@@ -0,0 +1,104 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+/**
+ * JS for Perspective Page
+ * features: display scores
+ */
+
+// DISPLAY SCORES
+
+/** displays the Perspective scores for the text in the "text-for-analysis" element  */
+/* eslint-disable no-unused-vars */
+async function displayScores() {
```

Comment:
Can you add `/*eslint-enable no-unused-vars*/` just below this line to re-enable the linting for the rest of the file (assuming the other functions are all used)?

(as done here: https://github.com/eslint/eslint/issues/1939#issuecomment-77225110)

Reply:
Yes I can.

Reply:
Done